Display/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM golang:latest
+EXPOSE 8000
+RUN apt update \
+    && apt install -y libzmq5-dev \
+                      python3-zmq \
+                      supervisor \
+                      python3-pip
+RUN go get github.com/pebbe/zmq4 && go get github.com/gorilla/websocket
+
+RUN mkdir -p /var/log/supervisor
+RUN mkdir -p /src/zmq_proxy/static
+WORKDIR /src/zmq_proxy
+COPY main.go /src/zmq_proxy
+COPY static /src/zmq_proxy/static
+RUN go build
+
+COPY requirements.txt /src/zmq_proxy
```

Comment:
If you go the route of copying individual files vs a whole directory, you have to make sure to copy new files when they get made, or else the docker image won't contain your changes.  As such, you should run tests against the docker image, which is the final deployable artifact, rather than the code itself.  (You may already be doing this, but I'm just starting the review).

Reply:
To confirm I'm understanding right, you're saying I should run the tests from within the docker image, yeah?

Reply:
no, I'm saying that when you start your behave tests, you should set up the environment to run the docker image, and query the docker image from the outside.  If you don't, then you run the risk of having a file outside of the docker that's not in the docker, and the tests passing, but the docker doesn't.